src/newsfeed/infrastructure/utils.py

```diff
@@ -0,0 +1,13 @@
+"""Utils module for infrastructure."""
+
+from urllib.parse import urlparse, parse_qsl
+
+
+def parse_redis_dsn(dsn):
```

Comment:
Why did you move this function to `utils` module?

Reply:
I moved it to a separate module in perspective to reuse in several places and to keep event storage clean.
Which place do you think will be better?

Reply:
This place is fine 👍